packages/fluent/scss/action-sheet/_layout.scss

```diff
@@ -1,3 +1,4 @@
+@use "sass:string";
```

Comment:
I believe this is not needed

Reply:
Yep. It's not. Previously, it was used for `string.unquote("max(...)")`, but then I remembered dart sass differs between css and math max.